WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -48,9 +49,16 @@ public CoinJoinManager(
 
 	public ImmutableDictionary<WalletId, ImmutableList<SmartCoin>> CoinsInCriticalPhase { get; set; } = ImmutableDictionary<WalletId, ImmutableList<SmartCoin>>.Empty;
 	public IWalletProvider WalletProvider { get; }
-	public IWasabiHttpClientFactory HttpClientFactory { get; }
-	public RoundStateUpdater RoundStatusUpdater { get; }
-	public CoinPrison CoinPrison { get; }
+	public WasabiHttpClientFactory? HttpClientFactory { get; }
+	public RoundStateUpdater? RoundStatusUpdater { get; }
+	public PersonCircuit? RoundStateUpdaterCircuit { get; }
+	public CoinPrison? CoinPrison { get; }
+
+	// A coordinator is configured if the backend URI is set to something other than the deprecated zkSNACKs' API endpoints.
+	public bool HasCoordinatorConfigured => HttpClientFactory is not null &&
+	                                        !(HttpClientFactory.BackendUriGetter is null ||
```

Comment:
I will make a new PR after this one to rename `BackendUriGetter` to `BaseUriGetter`, current naming makes no sense and is confusing